tests/foreman/cli/test_ping.py

```diff
@@ -56,3 +56,27 @@ def test_positive_ping():
         assert result.return_code == 0, 'Return code should be 0 if all services are ok'
     else:
         assert result.return_code != 0, 'Return code should not be 0 if any service is not ok'
+
+
+@pytest.mark.tier1
+def test_negative_ping_fail_status_code():
+    """Negative test to verify non-zero status code of ping fail
+
+    :id: 8f8675aa-df52-11eb-9353-b0a460e02491
+
+    :BZ: 1941240
+
+    :expectedresults: Hammer ping fails and returns non-zero(1) status code.
+
+    """
+
+    result = ssh.command(
+        "satellite-maintain service stop --only pulpcore-resource-manager.service"
+    ).return_code
+    assert result == 0
+    result = ssh.command("hammer ping").return_code
+    assert result == 1
+    result = ssh.command(
+        "satellite-maintain service start --only pulpcore-resource-manager.service"
+    ).return_code
+    assert result == 0
```

Comment:
Since all these commands are running on the satellite, I think you should use `default_sat` fixture here.
```
command = 'foo bar'
command_output = default_sat.execute(command)
```

Reply:
and then you can do assertion like `assert command_output.status == 0`.

Reply:
I'm inclined to move this to the destructive tier, because if the ping assertion fails or if the test fails to restart the service it could potentially affect other tests. @jameerpathan111 @JacobCallahan @akhil-jha what do you think?

Reply:
@pondrejk yes, we should move it to destructive tier.

Reply:
Added.